### PR TITLE
Update docs to v2 config

### PR DESCRIPTION
**Pull request by alculquicondor**
Config changed in #1163

### CODE REVIEW

**Comment by alculquicondor** [comment]
anything else here?

**Comment by mikebrow** [comment]
no just waiting for Lantau to get back as he had worked on this..

File: docs/config.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ for more information about containerd config.
 
 The explanation and default value of each configuration item are as follows:
 ```toml
```
Comment (Random-Liu) [inline docs/config.md]: Add `version = 2`. Version 2 is not the default, we need to explicitly specify it.

See https://github.com/containerd/containerd/pull/3337

Comment (mikebrow) [reply docs/config.md]: version 1 is the default if the field is not specified...

Comment (mikebrow) [reply docs/config.md]: Here's my suggested text: 

The version field in the config file specifies the config's version. If no version number is specified inside
 the config file then it is assumed to be a version 1 config and parsed as such. Use version = 2 to enable/access to new/current config fields.

Comment (alculquicondor) [reply docs/config.md]: Done.

**Pull request merged**